src/transaction/log_page_buffer.c

```diff
@@ -314,9 +314,7 @@ static int log_data_length = 0;
 
 LOG_LSA NULL_LSA = { NULL_PAGEID, NULL_OFFSET };
 
-#define LOG_ZIP_MIN_SIZE_TO_COMPRESS log_zip_min_size_to_compress
-
-static int log_zip_min_size_to_compress = 0;
+static int log_zip_min_size_to_compress = 255;
```

Comment:
@razvanradulescu 

Would you please rename it as `log_Zip_min_size_to_compress` to follow naming convention for globals?